Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -60,6 +60,7 @@ namespace PhysX
                         "Specifies the PhysX mesh collider asset for this PhysX collider component.")
                         ->Attribute(AZ_CRC_CE("EditButton"), "")
                         ->Attribute(AZ_CRC_CE("EditDescription"), "Open in Scene Settings")
+                        ->Attribute(AZ_CRC_CE("DisableEditButtonWheNoAssetSelected"), true)
```

Comment:
Would prefer we add this to the pre-defined constants in `Code\Framework\AzCore\AzCore\Serialization\EditContextConstants.inl` with a description, so it is more discoverable by users.

Reply:
I'm 50/50 on this one. While I'd like to have constant for the names, these attributes are only used by `PropertyAssetCtrl`, `EditContextConstants.inl` seems to have more generic names commonly used... I'm not sure that bringing all the EditButton attributes there is the right call. Since there are other attributes used like this, I think I'm going to let it as is for the moment.
Feel free to create a Github issue to create constants for all the attributes under EditContextConstants.inl with some kind of pattern. It could be a good first issue.

Reply:
nit - looks like a missing n at the end of the When in DisableEditButtonWheNoAssetSelected